docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,32 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
-
+    
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | When a voter can select only one candidate in a contest.                         |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | When a voter can select as many candidates as desired.                           |
```

Comment:
It adds something because not every enum value in our list has the pattern: "value" translates to "[value] voting."  This is only true for some of them, so it adds something to point out which ones.

Also, it might seem obvious to us, but not all readers necessarily know that "approval voting" is a "thing" (with a defined meaning, etc).  They might just think it is a shorthand word we picked.

I'm also okay with hyperlinking the text "Approval voting" to the Wikipedia entry.